src/Giraffe/ComputationExpressions.fs

```diff
@@ -1,30 +1,31 @@
-/// **Description**
-///
+/// <summary>
 /// A collection of F# computation expressions:
 ///
 /// `opt {}`: Enables control flow and binding of Option<'T> objects
 /// `res {}`: Enables control flow and binding of Result<'T, 'TError> objects
-///
+/// </summary>
 module Giraffe.ComputationExpressions
 
+/// <summary>
```

Comment:
moved the docs to the type - seems to make more sense but could be missing something